docs/pages/blog/mui-x-v5.3.0.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Introducing Row Grouping Feature'
+description: After more than a year of development on the DataGrid, the first premium feature is live.
+date: 2022-01-15T00:00:00.000Z
+authors:
+  ['oliviertassinari', 'm4theushw', 'flaviendelangle', 'DanailH', 'alexfauquette']
+card: true
+---
+
+<!-- <img src="/static/blog/2021/card.png" alt="" style="width: 100%; margin-bottom: 16px;" /> -->
+
+# A step to premium plan: Row Grouping
+
+A lot of you were asking about it, and we are pleased to announce that the premium plan is on the road.
+After an incredible year fully focused on improving the DataGrid, we are moving forward by releasing a first feature of the premium plan: the row grouping which is released in v5.3.0.
+Let me introduce this new feature.
+
+## Start navigating the data 🚢
+
+If you already use the pro plan, you probably know the tree data that allows your users to navigate in the hierarchy by opening/closing children of a row.
```

Comment:
Most of the pro users probably don't know it yet.
We may keep this wording to emphasize the importance of these kind of features
Or rephrase to something like

```suggestion
If you already use the pro plan, you may be familiar with the tree data which allows your users to navigate in the hierarchy by opening/closing children of a row.
```